app/sockets/service.py

```diff
@@ -101,7 +101,44 @@ def create_conversation(conversation_id, creator, participants):
                 result.append(participant)
                 
         return result
-    
+
+    @staticmethod
+    def remove_identifier(email):
```

Comment:
Maybe this should return `True` or `False` depending on whether or not the identifier was actually removed